src/test/java/com/google/api/codegen/testdata/csharp/csharp_library.baseline

```diff
@@ -5218,28 +5235,28 @@ namespace Google.Example.Library.V1.Tests
 
 // Generated code. DO NOT EDIT!
 
-using Google.Api.Gax;
-using Google.Api.Gax.Grpc;
+using gax = Google.Api.Gax;
+using gaxgrpc = Google.Api.Gax.Grpc;
 using Google.Cloud.Tagger.V1;
-using Google.LongRunning;
-using Google.Protobuf;
-using Google.Protobuf.WellKnownTypes;
+using lro = Google.LongRunning;
+using proto = Google.Protobuf;
+using protowkt = Google.Protobuf.WellKnownTypes;
 using Google.Tagger.V1;
```

Comment:
Why do we have two different namespaces for Tagger? Is that deliberate?

Reply:
This is not due to this PR, it had both imports before. I suspect this may be due to a bad configuration in the test library; but this can be followed up in a future PR.